### PR TITLE
Improve performance & extensibility

### PR DESCRIPTION
**Pull request by zh-plus**
1. Using list comprehension to improve performance.
2. Using switcher to improve extensibility.

### CODE REVIEW

File: NumpyXBench/operators/common_ops.py [hunk 1]
```diff
@@ -43,14 +52,11 @@ def get_forward_func(self):
...
-        elif self._backend in ['mxnet', 'mx']:
-            module = sys.modules['mxnet.numpy']
-        elif self._backend in ['pytorch', 'torch']:
-            module = sys.modules['torch']
-        else:
-            raise NotImplementedError("Backend not supported now!")
+        try:
+            module = sys.modules[module_switcher[self._backend]]
+        except ValueError:
+            raise NotImplementedError(f'Backend: {self._backend} not supported now!')
```
Comment (hgt312) [inline NumpyXBench/operators/common_ops.py]: ok, change to 'not support or backend not installed'

File: NumpyXBench/operators/common_ops.py [hunk 2]
```diff
@@ -6,12 +6,21 @@
...
 except ImportError:
     pass
+
 from jinja2 import Template
 
 __all__ = []
 
 common_op_list = ['add', 'subtract', 'multiply', 'divide']
 
+module_switcher = {
```
Comment (hgt312) [inline NumpyXBench/operators/common_ops.py]: add `common.py` in utils and move this to there

Comment (hgt312) [reply NumpyXBench/operators/common_ops.py]: also use it in `benchmark`

File: NumpyXBench/operators/common_ops.py (hunk 1 above)
Comment (zh-plus) [reply NumpyXBench/operators/common_ops.py]: Done

File: NumpyXBench/operators/common_ops.py (hunk 2 above)
Comment (zh-plus) [reply NumpyXBench/operators/common_ops.py]: Done

**Review by hgt312 — APPROVED** [review]

**Pull request merged**